src/concurrency/timestamp_ordering_transaction_manager.cpp

```diff
@@ -29,7 +29,7 @@ Spinlock *TimestampOrderingTransactionManager::GetSpinlockField(
     const storage::TileGroupHeader *const tile_group_header,
     const oid_t &tuple_id) {
   return (Spinlock *)(tile_group_header->GetReservedFieldRef(tuple_id) +
-                      LOCK_OFFSET);
```

Comment:
This looks like you just made formatting changes to this file. What logic did you actually change?